### DISCUSSION TITLE
Accessibility Option for GitHubs New Lists features

### DISCUSSION THREAD

**Discussion by FahimKamal** [community/community#41486] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think an option should be added to make a list item to be visible to everyone and only to me. For example, maybe I have created a list item where I placed some of my private repos that I don't want others to see.

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @FahimKamal! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Kdmcbee88** [reply]
> Hey there!  👋 
> 
> Thanks for posting in the GitHub Community, @FahimKamal! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
> 
> I've gone ahead and moved this to the correct category for you. Good luck!

Thank you

**Reply by IIIRoMIII** [reply]
G

**Comment by Rinppn9999** [comment]
It's up to you. Let's take it easy. 😄

**Comment by Dalheart** [comment]
Hello everyone

**Comment by jfullstackdev** [comment]
that list is visible but private repos added to it are not, so others will be seeing it as empty

**Comment by TalleA15** [comment]
Hello all
Needed an assistant because I'm novice 
here can some help me step by step so as to enable me acquire more nowledge  
on configuration

**Comment by TalleA15** [comment]
I needed an assistant can somebody instruct me on how to configure my Gut Ghub l happen to joint you by chance. By the way what is the lnportant of this Git Ghub

**Reply by jfullstackdev** [reply]
what do you want to configure on GitHub ? there are so many features of GitHub,

maybe your profile ? the Profile README ? 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme
